docs/getting_started/tutorial.md

```diff
@@ -870,6 +870,6 @@ Finally, we can update the `blog_page.html` template to display the categories:
 
 ## Where next
 
--   Read the Wagtail [topics](../topics/index) and [reference](../reference/index) documentation
--   Learn how to implement [StreamField](../topics/streamfield) for freeform page content
--   Browse through the [advanced topics](../advanced_topics/index) section and read [third-party tutorials](../advanced_topics/third_party_tutorials)
+-   Read the Wagtail [topics](../topics/index) and [reference](../reference/index) documentation.
+-   Learn how to implement [StreamField](../topics/streamfield) for freeform page content.
+-   Browse through the [advanced topics](../advanced_topics/index) section and read [third-party tutorials](../advanced_topics/third_party_tutorials).
```

Comment:
These lists do not need a period at the end

Reply:
Thanks for the feedback.